src/librustc_privacy/lib.rs

```diff
@@ -147,6 +150,7 @@ impl<'a, 'tcx> Visitor<'tcx> for EmbargoVisitor<'a, 'tcx> {
     }
 
     fn visit_item(&mut self, item: &'tcx hir::Item) {
+        debug!("Walked item {:?}", item);
```

Comment:
Usually this would be something like: `debug!("visit_item({:?})", item);`, so it's easier to see where it comes from.